### PR TITLE
Resize calendar in the TE edit flow to show all dates

### LINKED ISSUE #2777: Scrollability in time view calendar
## 🐛 Bug Report
On my Xiaomi 5A (running Android 7), there is a scrollbar in the calendar, so I can scroll to see the 6th week of a month.

On the random Samsung Galaxy J5 (SM-J510FN) (running Android 6) there is no scrollbar. There is also no separator line between the calendar and buttons below.

Xiaomi 5A / Samsung J5

screen shot 2018-07-31 at 12 21 36

## Expected behavior
This should work the same on all phones. Either be scrollable or big enough/adapt to the size of the calendar. Ping @pe0ny for intended behaviour.

## To reproduce
Check different phones/OSes(?), and check with a month that spans six weeks.

### PR DESCRIPTION
**Pull request by dshr**
## What's this?
Resizes the TE Edit calendars to fit 6 week months.

### Relationships
Closes #2777 and #2745

## Why do we want this?
So users can set any date they like.

## How is it done?
By changing the height of the calendar.

### Why not another way?
Tried making the pager view stretchy, but the child Date Picker always returned the same height. 

### Side effects
Might look a tiny bit weird on shorter months.

## Review hints
Comment out `TogglDroidDatePicker.cs:79` and `TogglDroidDatePicker.cs:82` and navigate to December or July 2017.

## :squid: Permissions
All yours!

### CODE REVIEW

**Comment by dshr** [comment]
@kornelijepetak can you check this on your emulator once again?

File: Toggl.Giskard/Fragments/SelectTimeFragment.cs [hunk 1]
```diff
@@ -155,7 +155,7 @@ private void setupDialogWindowPosition()
         {
             var window = Dialog.Window;
             var layoutParams = window.Attributes;
-            layoutParams.Gravity = GravityFlags.Top;
+            layoutParams.Gravity = GravityFlags.Center;
```
Comment (daividssilverio) [inline Toggl.Giskard/Fragments/SelectTimeFragment.cs]: It doesn't work without changing this part? The dialog was placed in the top of the screen intentionally. It was part of the design.

Comment (dshr) [reply Toggl.Giskard/Fragments/SelectTimeFragment.cs]: The designs in Zeplin were centered, so I figured the top alignment was just some legacy stuff. I'll change it back and try again.

As for really tiny devices — they're about 3-4% of the market, so I think we're okay shipping as is (after I switch the gravity back to `Top` and verify that everything works) and then do a proper refactor to get this working on tiny devices.

Comment (dshr) [reply Toggl.Giskard/Fragments/SelectTimeFragment.cs]: Set it back to `Top`, everything works fine now.

Comment (daividssilverio) [reply Toggl.Giskard/Fragments/SelectTimeFragment.cs]: o/
It's hard to find and Zeplin wasn't updated it seems. Pinging @pe0ny about that.
The PR that introduced that is this one: #2125

Comment (pe0ny) [reply Toggl.Giskard/Fragments/SelectTimeFragment.cs]: Yep, let's go with the solution Leez proposed in #2125

**Pull request merged**